pytorch_to_returnn/torch/nn/modules/variable.py

```diff
@@ -17,16 +17,25 @@ def __init__(self, param: Parameter):
     super(Variable, self).__init__()
     assert isinstance(param, Parameter)
     self.param = param
+    self.parent_mod = None
```

Comment:
Type should be documented.
```suggestion
    self.parent_mod = None  # type: Optional[Tuple[Module, str]]
```
But maybe this can directly be a param in `__init__`? That would be cleaner instead of setting it from outside.

Reply:
Yes, I added in as an argument to `__init__`.